### PR TITLE
identify tool child feature

### LINKED ISSUE #5904: [FEATURE] Show related (child) features in identify results tree
Original commit: https://github.com/qgis/QGIS/commit/94e484f2640597fc4220755bdfa5081818c05720 by nyalldawson

Under each identified parent feature, new groups will show containing
any related child features, showing their attributes, actions, and
allowing the user to directly open the feature form for the child feature.

Fixes #18634

### PR DESCRIPTION
**Pull request by lajunek**
There is an entry for the related child features in the identify results dialog now. Described the information that can be derived from this entry.

Goal: Precise information about the new identify results dialog.

Ticket(s): fix #5904 

- [ ] Backport to LTR documentation is required

### CODE REVIEW

**Comment by lajunek** [comment]
@havatv @DelazJ can you give an advice: how can I solve the HTML build problems?

File: docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -1693,7 +1693,17 @@ default it will display the following information:
...
   feature that has been clicked.
-
+* information about the related child feature if you defined a relation as described
+  in :ref:`vector_relations´:
+  
+  * the name of the relation
+  * the entry in reference field, e.g. the name of the related child feature
+  * **Actions**: Also for the child feature you can define actions
+    in the layer's properties dialog (see :ref:`actions_menu`) and the default
+    action is ``View feature form``.
```
Comment (DelazJ) [inline docs/user_manual/introduction/general_tools.rst:1703]: You do not define the actions here, they are just listed if of canvas scope afaict
```suggestion
  * **Actions**: lists actions defined in the layer's properties dialog (see :ref:`actions_menu`)
    and the default action is ``View feature form``.
```

**Review by DelazJ — COMMENTED** [review]
Can you widen the screenshot to make the whole label visible, please?
Thanks

**Comment by DelazJ** [comment]
> how can I solve the HTML build problems? Do we have a tutorial where one can learn about sphinx build errors?

https://docs.qgis.org/3.16/en/docs/documentation_guidelines/first_contribution.html#make-corrections
In a few words, click the `Details` next to the failing test and scroll (up) in the page that opens. You have to read the lines but Most of the time, it's a line longer than the others or different from its neighbours. There you find why the build failed and the line involved
Hope that helps.

**Pull request merged**